repeat.lic

```diff
@@ -0,0 +1,14 @@
+=begin
+  Documentation:
```

Comment:
the repeated items must be double quoted e.g. `;repeat "get potato from my backpack" "put potato in urn"`

Reply:
will add docs in separate PR